std/stdio.d

```diff
@@ -2242,6 +2242,42 @@ $(XREF file,readText)
         assert(blc.front is blc.front);
     }
 
+    /**
+    Creates an input range set up to parse one line at a time from the file
+    into a tuple.
+
+    Range primitives may throw $(D StdioException) on I/O error.
+
+    Params:
+        file = file handle to parse from
+        format = tuple record $(XREF format,formattedRead format)
```

Comment:
Try to use REF instead, it's a quite new, more generic replacement that replaces LREF, XREFn etc

https://github.com/D-Programming-Language/dlang.org/pull/1184